addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4496,20 +4496,21 @@ registry.Box = SnippetOptionWidget.extend({
      */
     _getDefaultShadow(type, shadowClass) {
         const el = document.createElement('div');
+        let shadow = 'none';
```

Comment:
Don't define that variable in the middle of the div creation code... either before or after. I think the best place is like just before the switch?